.solitarc.js

```diff
@@ -1,8 +1,8 @@
 const path = require("path");
-const programDir = path.join(__dirname, "..", "splitwave");
-const idlDir = path.join(__dirname, "idl");
-const sdkDir = path.join(__dirname, "src", "generated");
-const binaryInstallDir = path.join(__dirname, ".crates");
+const programDir = path.join(__dirname, "./programs/splitwave");
+const idlDir = path.join(__dirname, "./client/idl");
+const sdkDir = path.join(__dirname, "./client/src", "generated");
+const binaryInstallDir = path.join(__dirname, "./client/.crates");
```

Comment:
We use `app` directory for solita sdk and I don't think we need to change this solita configuration.